src/css-modules/CategoryTiles.module.css

```diff
@@ -11,16 +11,15 @@
 
 .tile {
   width: 100%;
-  
-  min-width: 107px;
-  min-height: 72px;
+  max-width: 107px;
```

Comment:
Looking good! The only thing is this. Now the filter chips do not have the same size. But this something we will have to address later with media queries. Since on my phone they look too large.